packages/grafana-ui/src/components/Forms/Form.story.tsx

```diff
@@ -145,25 +148,11 @@ export const DefaultValues = () => {
       switch: false,
     },
   ];
-  const [defaultsIdx, setDefaultsIdx] = useState(0);
 
-  return (
-    <>
-      {renderForm(defaultValues[defaultsIdx])}
-      <Button
-        onClick={() => {
-          setDefaultsIdx((s) => (s + 1) % 2);
-        }}
-        variant="secondary"
-      >
-        Change default values
-      </Button>
-    </>
-  );
```

Comment:
Is there a reason to remove this? If yes, then `defaultValues` array should probably be updated as well.

Reply:
yeah...noticed the `change default values` button doesn't have any effects on the form. The change in the state doesn't cause any re-rendering thus making the button ineffective.

Would proceed to remove the second object in the `defaultValues `array.
Thanks! @Clarity-89 🙌